client/odie/message/style.scss

```diff
@@ -123,9 +140,13 @@
 }
 
 .odie-sources-inline {
-	display: inline-flex;
-	padding: 0 8px;
+	display: inline;
+	padding: 0;
 	margin: 0;
+
+	.odie-sources-link {
+		padding: 0 8px;
```

Comment:
```suggestion
		padding: 0 4px;
```

IMHO, the 8px side padding is too much:
<img width="395" alt="Screenshot 2023-11-10 at 21 19 05" src="https://github.com/Automattic/wp-calypso/assets/3392497/145f874f-0e96-4296-a72b-fb1998deeef4">

4px looks more balanced on the sides:
<img width="399" alt="Screenshot 2023-11-10 at 21 20 18" src="https://github.com/Automattic/wp-calypso/assets/3392497/5450c281-9258-4871-a292-f2869145da8a">

Reply:
Thanks! Just updated